src/main/java/com/exasol/projectkeeper/AbstractProjectKeeperMojo.java

```diff
@@ -50,7 +60,8 @@
         return List.of(new ProjectFilesValidator(enabledModules, this.project.getBasedir(), excludedFilesMatcher),
                 new PomFileValidator(enabledModules, this.project.getModel().getPomFile()),
                 new ChangesFileValidator(this.project.getVersion(), this.project.getName(),
```

Comment:
Please, extract a method

Reply:
I still think it's more readable and easier to modify it they have separated methods, or at least variables. But you can leave them as it is, no problems